src/main/java/frc/robot/Controls.java

```diff
@@ -100,12 +100,24 @@ private void configureElevatorBindings() {
     // Controls binding goes here
     operatorController.leftTrigger().whileTrue(s.elevatorSubsystem.goUpPower());
     operatorController.rightTrigger().whileTrue(s.elevatorSubsystem.goDownPower());
-    operatorController.y().onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_FOUR_POS));
-    operatorController.x().onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_THREE_POS));
-    operatorController.b().onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_TWO_POS));
-    operatorController.a().onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_ONE_POS));
+    operatorController
+        .y()
+        .onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_FOUR_POS).withName("L4"));
+    operatorController
+        .x()
+        .onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_THREE_POS).withName("L3"));
+    operatorController
+        .b()
+        .onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_TWO_POS).withName("L2"));
+    operatorController
+        .a()
+        .onTrue(s.elevatorSubsystem.setLevel(ElevatorSubsystem.LEVEL_ONE_POS).withName("L1"));
```

Comment:
For the command names, please prefix with `Elevator`, such as `ElevatorL4` and `ElevatorIntakePos`. That'll disambiguate this from the 'L4' preset command I expect will also exist for the arm/elbow.